test/app/lib/crypto/helpers.ts

```diff
@@ -0,0 +1,20 @@
+import {ExtendedKey} from "../../../../app/lib/crypto/key/key"
+import {fromEntropy} from "../../../../app/lib/crypto/seed"
+import * as PrivateKey from "../../../../app/lib/crypto/key/privateKey"
+
+/**
+ * Helper for testing
```

Comment:
I'm a bit reluctant of including test helpers inside the library itself. Yet I'm not completely against it and can't figure out a better option in this case (moving this into the tests folders may require us to create a `helpers` directory that should be ignored by jest).